pydatastructs/graphs/adjacency_matrix.py

```diff
@@ -63,7 +63,7 @@ def add_edge(self, source, target, cost=None):
                 GraphEdge(self.__getattribute__(source),
                           self.__getattribute__(target),
                           cost)
-
+            
```

Comment:
The build is failing because of code quality.
Just remove the extra spaces here and use newline only.

Reply:
```suggestion
```